src/config.ts

```diff
@@ -63,6 +63,7 @@ export async function makeConfig(): Promise<Config> {
       target: parseArray(getInput('target')),
       targetDependents: parseBoolean(getInput('target-dependents')),
       editCommentOnPr: parseBoolean(getInput('edit-pr-comment')),
+      userAgent: "pulumi/action@v3"
```

Comment:
Should probably add `userAgent` to the run-type in the same file (`options`). It's probably good enough with an `rt.Literal('pulumi/actions@v3')`.

Reply:
Fixed and added the userAgent to the tests